### ISSUE TITLE
Differences of dates created unequally

### ISSUE THREAD

**Issue by mimischi** [date-fns/date-fns#1424]
I am trying to calculate the differnce between two dates. One is created with explicit date/time values via `new Date(2019, 9, 14, 10, 0)`. The other one is created without passing any arguments via `new Date()`. Giving these two date objects as arguments to any `distanceIn*` method will results in an unexpected result.

Here is an example: https://runkit.com/mimischi/5d7cb212e310710013b35354

Is there something I don't understand about how JavaScript handles dates?

## TL;DR

I want to do

```
const someDate = new Date(2019, 9, 14, 10, 0);
dateFns.differenceInMinutes(new Date(), someDate);
```

but this does not work.

**Comment by mimischi** [comment]
Well, I forgot to account for the zero indexed month in JavaScript. I knew that the difference of `-40k`-something reminded me of something.. This works:

```
const someDate = new Date(2019, 9 - 1, 14, 10, 0);
dateFns.differenceInMinutes(new Date(), someDate);
```

**Comment by dkozickis** [comment]
Thank you for reporting this and for figuring it out :) 
Closing.
